scene/gui/text_edit.cpp

```diff
@@ -634,6 +634,12 @@ void TextEdit::_notification(int p_what) {
 			if (draw_minimap) {
 				xmargin_end -= minimap_width;
 			}
+
+			// Use the local variable to adjust for increased scrollbar touch area.
+			if (increase_scrollbar_touch_area) {
+				xmargin_end -= 50; // Hardcoded value matching the increased touch area size.
```

Comment:
Instead of hardcoding the value, this should use the v scroll bar's width instead.